noted/notes/v1/groups.proto

```diff
@@ -174,17 +174,29 @@ service GroupsAPI {
     rpc ListConversationMessages(ListConversationMessagesRequest) returns (ListConversationMessagesResponse) {}
 
     // Must be group member. generated_by_account_id defaults to the authenticated user.
-    rpc GenerateInviteLink(GenerateInviteLinkRequest) returns (GenerateInviteLinkResponse) {}
+    rpc GenerateInviteLink(GenerateInviteLinkRequest) returns (GenerateInviteLinkResponse) {
+        option (google.api.http) = {
+            post: "/groups/{group_id}/inviteLinks",
```

Comment:
Maybe missing a `body: "*"`? I don't know maybe it's on by default but worth checking.

Reply:
I have no idea for this context cause `group_id` is already filled in the URL and the only thing in GenerateInviteLinkRequest is `group_id` :(
For what I saw, let's say for `GET` requests, when the params are filled out in the URL, there's no `body` option t fill (i mean yeah it doesn't make sense for GET requests anyway) but I think if we fill out every field of the Request object we don't have to specify the body field

wdyt?

Reply:
Oh my bad I thought there was more stuff in `GenerateInviteLinkRequest`. My bad! Merging.